tasks/src/main/scala/dagr/tasks/gatk/GenotypeGvcfs.scala

```diff
@@ -50,9 +50,18 @@ class GenotypeGvcfs private (ref: PathToFasta,
                              val dbSnpVcf: Option[PathToVcf] = None)
   extends GatkTask("GenotypeGVCFs", ref, intervals=intervals) {
 
+  require(gvcfs.length == 1 || gatkMajorVersion < 4, "GenotypeGVCFs only supports one GVCF at a time with GATK version 4+.")
+
   override protected def addWalkerArgs(buffer: ListBuffer[Any]): Unit = {
+    // Args that are common to all versions
     dbSnpVcf.foreach(v => buffer.append("--dbsnp", v.toAbsolutePath.toString))
     gvcfs.foreach(gvcf => buffer.append("-V", gvcf.toAbsolutePath.toString))
-    buffer.append("-o", vcf.toAbsolutePath.toString)
+
+    gatkMajorVersion match {
+      case n if n < 4  =>
+        buffer.append("--out", vcf.toAbsolutePath.toString)
+      case n if n >= 4 =>
+        buffer.append("--output", vcf.toAbsolutePath.toString)
```

Comment:
Do we need `.toString` here and elsewhere since we call it later?

Reply:
I have no idea why there were `.toString`s there in the first place.  Removed.